client/reader/sidebar/index.jsx

```diff
@@ -254,22 +235,51 @@ const ReaderSidebar = React.createClass( {
 					onTagExists={ this.highlightNewTag }
 					currentTag={ this.state.currentTag } />
 
-				</SidebarRegion>
+			</SidebarRegion>
 
-				{ this.renderAppPromo() }
+			{ this.props.shouldRenderAppPromo &&
+				<div className="sidebar__AppPromo">
```

Comment:
While the prefix is now correct, latter half of the class name should be lower-case and dash-delimited, i.e. `sidebar__app-promo`.